tests/operators/test_plots.py

```diff
@@ -121,3 +121,29 @@ def test_plot_line_series_too_many_dimensions(cube, tmp_working_dir):
     """Error when cube has more than one dimension."""
     with pytest.raises(ValueError):
         plot.plot_line_series(cube)
+
+
+def test_plot_vertical_line_series(monkeypatch, tmp_path):
+    """Save a vertical line series plot."""
+    cube = read.read_cube(
+        "tests/test_data/air_temperature_vertical_profile_as_series.nc"
+    )
+    monkeypatch.chdir(tmp_path)
```

Comment:
```suggestion
def test_plot_vertical_line_series(tmp_working_dir):
    """Save a vertical line series plot."""
    cube = read.read_cube(
        "tests/test_data/air_temperature_vertical_profile_as_series.nc"
    )
```
`tmp_working_dir` is a fixture I've setup that gives you a temporary working directory for the duration of the test.